docs/2_3_common_states.adoc

```diff
@@ -1,63 +1,68 @@
 === State Machine and Semantics [[common-state-machine]]
 
+To define allowed calling sequences of FMI functions, state machines are used.
 All FMI interface types share a number of states in their respective state machines.
-This chapter describes these common modes.
-FMI specific state-machine modes will be described in their respective chapters.
+This chapter describes these common states used in at least two of the interface types.
+FMI specific state-machine states will be described in their respective chapters.
+Each state of the state machine corresponds to a certain phase of a simulation.
 
 .Common calling sequence for C functions of common states for all three FMI types.
```

Comment:
The title says: common states, the state diagram is obviously not complete.
The way it is shown here makes it simple to export from the diagram editor and since the arrows do not match up perfectly with the grey state for all interface types, the grey state could cause confusion (see how to enter Event Mode).